.github/actions/terraform-setup/action.yml

```diff
@@ -207,6 +220,18 @@ runs:
       working-directory: ${{ inputs.working-directory }}
       shell: bash
 
+    - name: Set fluent-bit-ci secrets
+      run: |
+        [[ -n "$FLUENT_BIT_CI_OPENSEARCH_AWS_ACCESS_ID" ]] && echo "fluent-bit-ci-opensearch-aws-access-id = \"$FLUENT_BIT_CI_OPENSEARCH_AWS_ACCESS_ID\"" >> default.auto.tfvars
+        [[ -n "$FLUENT_BIT_CI_OPENSEARCH_AWS_SECRET_KEY" ]]  && echo "fluent-bit-ci-opensearch-aws-secret-key = \"$FLUENT_BIT_CI_OPENSEARCH_AWS_SECRET_KEY\"" >> default.auto.tfvars
```

Comment:
You could use single quotes and the `${{ inputs.X }}` value directly to prevent need for escaping?